.github/workflows/workflow-httpstan.yml

```diff
@@ -0,0 +1,341 @@
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir tox
+      - name: Run tox
+        env:
+          TOXENV: mypy
+        run: |
+          python -m tox
+
+
+  #### TOX ####
+  tox-docs:
+    name: docs
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir tox
+      - name: Run tox
+        env:
+          TOXENV: docs
+        run: |
+          python -m tox
+      - uses: actions/upload-artifact@v2
+        with:
+          name: httpstan_docs
+          path: build/
+
+  #### TEST ####
+  pytest:
+    name: Run tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.7, 3.8]
+        compiler: [gcc, clang]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: debug packages
+        run: |
+          python -v
+          which python
+          make --version
+          which make
+          gcc --version
+          which gcc
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          make
+      - name: Install httpstan
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m poetry install -v
+      - name: Run tests
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m poetry run pytest -s -v --cov=httpstan --cov-fail-under=93 --forked tests
+      - name: Upload coverage
+        if: success()
+        run: |
+          bash <(curl -s https://codecov.io/bash)
+
+
+  #### WHEEL (manylinux2014) ####
+  publish_manylinux2014:
+    name: manylinux2014 wheel
+    needs: [pytest, tox-flake8, tox-format, tox-mypy, tox-docs]
+    runs-on: ${{ matrix.os }}
+    container:
+      image: quay.io/pypa/manylinux2014_x86_64:latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [cp37-cp37m, cp38-cp38]
+        compiler: [clang]
```

Comment:
Why are we using clang here?

Reply:
I just needed to update the compiler and used only clang.

The gcc update is similar.

Which one should we use (for manylinux2014)?

Reply:
I think we should match everything to the configuration we use in httpstan-wheels. We use manylinux2014 with few changes. I don't think we change the compiler at all.

Reply:
We don't but I think multibuild does. Because the default compiler is 4.x?

Do we need C++14 for math?

I would have used multibuild, but I think it currently does not support Actions (there is an open issue, and I don't know if we could implement that actions setup for it).

Action tools need to wrap the tool with Docker or JS.

The thing I'm not sure / happy is that the procedure for macOS wheel might not be compatible with older macOS. Not sure how multibuild handles that.